gallery/tutorials/configuration.py

```diff
@@ -120,6 +120,60 @@
 # You can also resolve the config in code
 print(aspire.config.dump())
 
+# %%
+# Logging preferences
+# ---------
+# The "Logging" section contains options for controlling the verbosity and destination of log messages generated by ASPIRE.
+#
+# By default, the ``log_dir`` is a directory called ``logs`` within the current working directory from which ASPIRE was invoked, but
+# it can be changed to an absolute path (for instance, to go to the same folder as a custom configuration file stored elsewhere, such
+# as in the example above):
+#
+#    .. code-block:: yaml
+#
+#      logging:
+#          log_dir=/tmp/my_proj/aspire_logs
+#
+# ASPIRE directs output streams to both the console and to a log file during each session. The logging verbosity for both of these
+# can be specified individually via ``console_level`` and ``log_file_level``. These levels are passed through to Python's logging
+# system, so the labels are the same: ``DEBUG, INFO, WARNING, ERROR, CRITICAL``. In general, it may make sense to have the log file
```

Comment:
Maybe separate quotes for each label: ``DEBUG``, ``INFO``, ``WARNING``, ``ERROR``, ``CRITICAL``